### ISSUE TITLE
Drop down - WCAG issue

### ISSUE THREAD

**Issue by UiTeamIntegra** [Pearson-Higher-Ed/elements-sdk#171]
When accessing the drop down using voice readers, we could navigate using arrow keys but couldn't navigate using 'tab' key. [https://pearson-higher-ed.github.io/elements-sdk/#/dropdown](url)

**Comment by StommePoes** [comment]
Hi,
this component is following the keyboard pattern for Menu Buttons and Menus: https://www.w3.org/TR/wai-aria-practices-1.1/#menu

Similar to native &lt;select&gt; elements, the options within the opened menu are not meant to be Tabbed to. The widget as a whole must be a single Tab stop.

I am assuming you mean, you *can* Tab to the button examples ("text", "Button text" controls), but that once the menu is open, you cannot Tab between those options. Do I understand correctly, or are you unable to Tab to the item itself at all?
